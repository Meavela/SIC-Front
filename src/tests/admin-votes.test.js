import React from 'react';
import renderer from 'react-test-renderer';
import AdminVotes from '../components/admin-votes';
import { BrowserRouter } from 'react-router-dom';

it('admin votes page correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
            <AdminVotes />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });