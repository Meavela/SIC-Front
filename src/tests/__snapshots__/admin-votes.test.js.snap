// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`admin votes page correctly 1`] = `
Array [
  <a
    href="/admin"
    onClick={[Function]}
  >
    â¬… Back
  </a>,
  <div>
    <div
      className="row"
    >
      <div
        className="col-md-4 col-6"
      >
        <div
          className="list-group"
          onKeyDown={[Function]}
          style={
            Object {
              "marginTop": 200,
            }
          }
        />
      </div>
      <div
        className="col"
      >
        <div />
      </div>
    </div>
  </div>,
  <div>
    <h3>
      Ajouter une option de vote
    </h3>
    <form
      className=""
    >
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="formBasicUsername"
        >
          Titre :
        </label>
        <input
          className="form-control"
          id="formBasicUsername"
          onChange={[Function]}
          type="text"
        />
        <small
          className="text-muted form-text"
        />
      </div>
      <div
        className="submitButton form-group"
      >
        <button
          className="btn btn-primary"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          Create
        </button>
      </div>
    </form>
  </div>,
]
`;
